CHANGES.rst

```diff
@@ -2,9 +2,11 @@ Changes by Version
 ==================
 
 
-0.16.4 (unreleased)
+0.17.0 (unreleased)
 -------------------
 
+- Add ``tchannel.thrift.load`` to read and compile Thrift files at startup.
```

Comment:
Wording. Try:

It's now possible to load Thrift idl files directly with ``tchannel.thrift.load`` - this means the generation step using Apache Thrift's cli can be skipped entirely. Check the examples folder for usage.